Campaign/management/commands/ExportSystemScoresToCSV.py

```diff
@@ -26,6 +26,9 @@
     (PairwiseAssessmentTask, PairwiseAssessmentResult),
 )
 
+import csv
```

Comment:
These imports would be moved up by `reorder-python-imports`, so why not move them there in the first place?

Reply:
Fixed, thanks!